overview/default.nix

```diff
@@ -262,7 +262,7 @@ let
         --from=markdown+raw_html \
         --to=html \
         --standalone \
-        --css="/style.css" \
+        --css="./style.css" \
```

Comment:
This breaks css for project URLs: http://localhost:8080/project/Nitrokey-3.html

Reply:
Ugh... this needs more thought. But absolute URLs bad because you can't use the styles locally without web server shenanigans as it will think it's `file:///style.css` rather than the store path.

Reply:
I'd advise against opening stuff as `file://` in general. It breaks stuff in weird ways and doesn't even behave the same way across browsers. Instead we should put a simple web server into shell.nix, something like `darkttpd` that works out of the box with no config required.